src/net_processing.cpp

```diff
@@ -2267,6 +2355,29 @@ static void ProcessGetCFCheckPt(CNode& peer, CDataStream& vRecv, const CChainPar
     connman.PushMessage(&peer, std::move(msg));
 }
 
+/**
+ * Announce transactions a peer is missing after reconciliation is done.
+ * No need to add transactions to peer's filter or do checks
+ * because it was already done when adding to the reconciliation set.
+ */
+void static AnnounceTxs(const std::vector<uint256>& remote_missing_wtxids, CNode& pto, const CNetMsgMaker& msgMaker, CConnman& connman)
+{
+    if (remote_missing_wtxids.size() != 0) {
+        std::vector<CInv> remote_missing_invs;
```

Comment:
Perhaps reserve max(size of remote_missing_wtxids, MAX_INV_SIZE) to avoid reallocations.

Reply:
I think this should be min, because the vector never exceeds MAX_INV_SIZE.

Reply:
yes, you're right. Should be min.